dist/assets/css/main.css

```diff
@@ -1414,6 +1428,19 @@ footer #copyright p a:hover {
           flex-direction: column;
   margin: auto;
 }
+#article main section div div {
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: row;
```

Comment:
That code above isn't in my code in atom, but I don't make a habit of adding those prefixes to my code, I barely understand them lol